cmd/Makefile.am

```diff
@@ -372,9 +376,11 @@ install-exec-local::
 ##
 
 libexec_PROGRAMS += snap-discard-ns/snap-discard-ns
+if HAVE_RST2MAN
```

Comment:
Why adding something to CLEANFILES which will never exist when HAVE_RST2MAN isn't true?